README.md

```diff
@@ -315,16 +266,260 @@ For example:
   - `fields, fields->>'c2', sqrt((fields->>'c1')::int)`: function `sqrt()` is not pushed down.
   - `fields, sqrt((fields->>'c1')::int)`: function `sqrt()` is not pushed down.
   - `fields->>'c2', sqrt((fields->>'c1')::int)`: there is no fields jsonb column, so function `sqrt()` is pushed down.
+  
+Also see [Limitations](#limitations).  
+
+Supported platforms
+-------------------
 
-## Contributing
+`influxdb_fdw` was developed on Linux, and should run on any
+reasonably POSIX-compliant system.
+
+`influxdb_fdw` is designed to be compatible with PostgreSQL 10 ~ 14.
+
+Go version should be 1.10.4 or later.
+
+Installation
+------------
+### Prerequisites
+
+Install InfluxDB Go client library
+<pre>
+go get github.com/influxdata/influxdb1-client/v2
+</pre>
+
+### Source installation
+
+Add a directory of pg_config to PATH and build and install influxdb_fdw.
+<pre>
+make USE_PGXS=1 with_llvm=no
+make install USE_PGXS=1 with_llvm=no
+</pre>
+with_llvm=no is necessary to disable llvm bit code generation when PostgreSQL is configured with --with-llvm because influxdb_fdw use go code and cannot be compiled to llvm bit code.
+
+If you want to build influxdb_fdw in a source tree of PostgreSQL instead, use
+<pre>
+make with_llvm=no
+make install  with_llvm=no
+</pre>
+
+Usage
+-----
+
+## CREATE SERVER options
+
+`influxdb_fdw` accepts the following options via the `CREATE SERVER` command:
+
+- **dbname** as *string*, optional
+
+- **host** as *string*, optional
+
+- **port** as *integer*, optional
+
+
+## CREATE USER MAPPING options
+
+`influxdb_fdw` accepts the following options via the `CREATE USER MAPPING`
+command:
+
+- **user** as *string*, no default
+
+  The user credential to connect to InfluxDB.
+
+- **password** as *string*, no default
+
+  The password credential to connect to InfluxDB.
+
+
+## CREATE FOREIGN TABLE options
+
+`influxdb_fdw` accepts the following table-level options via the
+`CREATE FOREIGN TABLE` command.
+
+- **tags** as *string*, optional, no default
+    
+- **table** as *string*, optional, no default
+
+- **schemaless** as *boolean*, optional, default `false`
+  
+  Enable schemaless mode.
+
+## IMPORT FOREIGN SCHEMA options
```

Comment:
Following options are supported:
* **host** - the address used to connect to InfluxDB server.
* **port** - the port used to connect to InfluxDB server.
* **dbname** - target database name
* **retention_policy** - retention policy of target database (default empty ``).
* **user** - username for V1 basic authentication.
* **password** - password for V1 basic authentication.
* **auth_token** - token for V2 Token authentication.
* **version** - InfluxDB server version which to connect to (Only `1` or `2`). If not, InfluxDB FDW will try to connect to InfluxDB V2 first. If unsuccessful, it will try to connect to InfluxDB V1. If it is still unsuccessful, error will be raised.

Reply:
This look like C-styled list of options from C-code. For usage context I have documented this options in different places in USAGE section. Thanks, @weiting1lee !

Reply:
Yes, Usage section seems perfect.
Thank you.